helpers/members/banMember.ts

```diff
@@ -6,19 +6,20 @@ export async function banMember(bot: Bot, guildId: bigint, id: bigint, options?:
     bot.rest,
     "PUT",
     bot.constants.routes.GUILD_BAN(guildId, id),
-    options
-      ? {
-        delete_message_days: options.deleteMessageDays,
-        reason: options.reason,
-      }
-      : {},
+    {
+      delete_message_days: options?.deleteMessageDays,
+      delete_message_seconds: options?.deleteMessageSeconds,
+      reason: options?.reason,
+    },
   );
 }
 
 /** https://discord.com/developers/docs/resources/guild#create-guild-ban */
 export interface CreateGuildBan {
-  /** Number of days to delete messages for (0-7) */
+  /** Number of days to delete messages for (0-7) (deprecated) */
```

Comment:
If this is really deprecated just remove it, in dd we don't keep things that are going to be killed anyway